x-pack/plugin/esql/qa/testFixtures/src/main/resources/math.csv-spec

```diff
@@ -1379,6 +1379,20 @@ d:double | c:double
 -0.0     | -0.0
 ;
 
+hypot
+required_capability: fn_hypot
+// tag::hypot[]
+ROW a = 3.0, b = 4.0
+| EVAL c = hypot(a, b)
+// end::hypot[]
+;
+
+// tag::hypot-result[]
+a:double | b:double | c:double
+3.0      | 4.0      | 5.0
+// end::hypot-result[]
+;
+
```

Comment:
I would add some extra cases:
- Having `null` in one and both parameters
- Another one using `FROM` instead of `ROW`, as they work differently (`ROW` may get folded as it's all constants, for example)

Reply:
employees also has some numeric data types - integer, double, long etc.,  if you are looking for data types besides unsigned long. They are usable, although a little nonsensical too.

Reply:
++. It doesn't really matter if they make sense IMO, as long as they come from an index. For interesting values, there's already the function tests